endpoints/authorized.py

```
@@ -0,0 +1,39 @@
+        "model": UnauthorizedResponse,
+    },
+    403: {
+        "description": "User is not authorized",
+        "model": ForbiddenResponse,
+    },
+}
+
+
+@router.post("/authorized", responses=authorized_responses)
+async def authorized_endpoint_handler(
+    auth: Any = Depends(auth_dependency),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)